### DISCUSSION TITLE
Grist self hosted error

### DISCUSSION THREAD

**Discussion by CHUYKOPS** [outline/outline#4718] [Self Hosting] [answered]
Hey. 

We have self hosted Grist but when i try to add link it gives an error

![image](https://user-images.githubusercontent.com/88129934/212545458-4dcc832c-8dc8-4304-ab1a-0bc7a852c3c9.png)

How we can solve that? 

Many thanks!

**Comment by tommoor** [comment, marked as answer]
Grist embed only supports the cloud-hosted version at this time I'm afraid, someone needs to build support for the self-hosted version in a similar manner to the Draw.io integration.
